app/forms/steps/client/benefit_check_result_form.rb

```diff
@@ -2,17 +2,31 @@ module Steps
   module Client
     class BenefitCheckResultForm < Steps::BaseFormObject
       include Steps::HasOneAssociation
+      attribute :passporting_benefit, :value_object, source: YesNoAnswer
```

Comment:
I think the validation is missing in this form, meaning a provider can continue without selecting any radio option.